haskell/src/Lorentz/Contracts/Spec/FA2Interface.hs

```diff
@@ -167,14 +161,23 @@ type IsOperatorParam = View ("operator" :! OperatorParam) IsOperatorResponse
 -- solve it.
 -- TODO : https://gitlab.com/morley-framework/morley/-/issues/159
 data Parameter
-  = Transfer TransferParams
-  | Balance_of BalanceRequestParams
-  | Token_metadata TokenMetaDataParam
+  = Balance_of BalanceRequestParams
+  | Is_operator IsOperatorParam
   | Token_metadata_registry TokenMetadataRegistryParam
+  | Transfer TransferParams
```

Comment:
Is the ordering of constructors relevant here? If that is the case, shouldn't we leave a comment stating the same?

Reply:
I see that you have left the comment on the main param, so I guess it is fine.